.gitignore

```diff
@@ -5,3 +5,4 @@
 # From Tiny Swords pack
 .DS_Store
 .aseprite
+.vscode/settings.json
```

Comment:
This is unrelated to the rest of the change and should be its own commit.

Reply:
Sure. I can also see if I can exclude that file locally since maybe not everyone is using VS Code

Reply:
I think it's fine to add it, but as a Git pedant I prefer it as a separate commit :)